labgrid/driver/fastbootdriver.py

```diff
@@ -43,7 +45,10 @@ def on_deactivate(self):
     @Driver.check_active
     @step(title='call', args=['args'])
     def __call__(self, *args):
-        subprocess.check_call(self._get_fastboot_prefix() + list(args))
+        cmd = self._get_fastboot_prefix() + list(args)
+        self.logger.debug(' '.join(cmd))
+        stdout = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+        self.logger.debug(stdout)
```

Comment:
Hmm. Currently, if fastboot fails and you're not running in pytest with stdout capture, you just have the error message from fastboot on the console. That's gone with your change. Do we want to pass the output to a higher log level if an exception happens?

Reply:
Why is stdout output gone with my change? The exception should contain that, see my commit message.